crypto/src/main/java/net/corda/v5/crypto/KeyUtils.java

```diff
@@ -75,18 +74,15 @@ public static boolean isKeyInSet(@NotNull PublicKey key, @NotNull Iterable<Publi
      * check fulfilment against a set of keys, without having to handle simple and composite keys separately (that is, this is
      * polymorphic).
      * 
-     * @param firstKey  The key with the requirements.
+     * @param key  The key with the requirements.
      * @param otherKeys The key to check whether requirements are fulfilled.
      */
-    public static boolean isKeyFulfilledBy(@NotNull PublicKey firstKey, @NotNull Iterable<PublicKey> otherKeys) {
-        if (firstKey instanceof CompositeKey) {
-            CompositeKey firstKeyComposite = (CompositeKey) firstKey;
+    public static boolean isKeyFulfilledByKeys(@NotNull PublicKey key, @NotNull Set<PublicKey> otherKeys) {
```

Comment:
Similar, the second "Keys" doesn't add anything to the function name.

Reply:
Reverted name change